packages/components/datagrid/src/components/osds-datagrid/core/controller.ts

```diff
@@ -17,8 +17,9 @@ class OdsDatagridController {
             field: column.field,
             headerSort: column.isSortable ?? false,
             headerHozAlign: 'center',
+            vertAlign: 'middle',
             titleFormatter: (cell: CellComponent) => this.getOdsText(cell.getValue(), ODS_TEXT_SIZE._500),
-            formatter: (cell: CellComponent) => this.getOdsText(cell.getValue(), ODS_TEXT_SIZE._400),
+            formatter: (cell: CellComponent) => column.formatter?.(cell.getValue()) ?? this.getOdsText(cell.getValue(), ODS_TEXT_SIZE._400),
```

Comment:
would be nice to add a dedicated sandbox page with a very large datagrid (10000+ rows) to ensure we don't have any perf issue